modules/virt-runbook-lowvirtoperatorcount.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-runbooks.adoc
+
+:_content-type: REFERENCE
+[id="virt-runbook-lowvirtoperatorcount_{context}"]
+= LowVirtOperatorCount
+
+// Edited by Jiří Herrmann, 8 Nov 2022
+
+[discrete]
+[id="meaning-lowvirtoperatorcount_{context}"]
+== Meaning
+
+This alert fires when only one `virt-operator` pod in a `Ready` state has
+been running for the last 60 minutes.
+
+The `virt-operator` is the first Operator to start in a cluster. Its primary
+responsibilities include the following:
+
+* Installing, live-updating, and live-upgrading a cluster
+* Monitoring the lifecycle of top-level controllers, such as `virt-controller`,
+`virt-handler`, `virt-launcher`, and managing their reconciliation
+* Certain cluster-wide tasks, such as certificate rotation and infrastructure
+management
+
+[discrete]
+[id="impact-lowvirtoperatorcount_{context}"]
+== Impact
+
+The `virt-operator` cannot provide high availability (HA) for the deployment.
+HA requires two or more `virt-operator` pods in a `Ready` state. The default
+deployment is two pods.
+
+The `virt-operator` is not directly responsible for virtual machines (VMs)
+in the cluster. Therefore, its decreased availability does not significantly
+affect VM workloads.
+
+[discrete]
+[id="diagnosis-lowvirtoperatorcount_{context}"]
+== Diagnosis
+
+. Set the `NAMESPACE` environment variable:
++
+[source,terminal]
+----
+$ export NAMESPACE="$(oc get kubevirt -A \
+  -o custom-columns="":.metadata.namespace)"
+----
+
+. Check the states of the `virt-operator` pods:
++
+[source,terminal]
+----
+$ oc -n $NAMESPACE get pods -l kubevirt.io=virt-operator
+----
+
+. Review the logs of the affected `virt-operator` pods:
++
+[source,terminal]
+----
+$ oc -n $NAMESPACE logs <virt-operator>
+----
+
+. Obtain the details of the affected `virt-operator` pods:
++
+[source,terminal]
+----
+$ oc -n $NAMESPACE describe pod <virt-operator>
+----
+
+[discrete]
+[id="mitigation-lowvirtoperatorcount_{context}"]
+== Mitigation
+
+Based on the information obtained during Diagnosis, try to find the cause
```

Comment:
I get that this is try to point to the "Diagnosis" heading above, but it's bothering me that it's capitalized. It seems like it should be lowercased. This is used throughout the modules, if you do update.